### PR TITLE
Math-Magicians- Components

### PR DESCRIPTION
**Pull request by nati2323**
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches this design:

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+function App() {
+  return (
+    <>
+      <Display />
+      <ButtonPanel />
+    </>
+  );
+}
+
```
Comment (ishimwezachee) [inline src/components/App.js:12]: - [ ]  Your App.js component should be class-based component, and it should import the calculator.js file that contains the codes of the design.  Please follow the provided guideline and implement the requirements as suggested.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <div>
+      <h1>{ buttonName }</h1>
+    </div>
+  );
+}
+
+Button.propTypes = {
+  buttonName: PropTypes.string.isRequired,
+};
```
Comment (ishimwezachee) [inline src/components/Button.js:14]: - [ ] Please implement the project using Class base components as it is suggested by the requirements. 
 image

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <Display />
+      <ButtonPanel />
+    </>
```
Comment (ishimwezachee) [inline src/components/App.js:9]: - [ ]  Your design should match to the one provided from the requirements. Please re-read carreful the requirements and follow  the guideline for you to  got to the next milestone 

You design
image

Provided design 

image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @nati2323 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
PR is  well detailed 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @nati2323, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- The PR title and description looks professional✔️
- No linter errors✔️
- You have a descriptive readme file
✔️
 ## Required Changes ♻️

- [ ] Kindly work on the requested changes by the previous reviewer, currently, all the components are functional based which is the opposite of the project requirements, the design too should be addressed.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.js (hunk 3 above)
Comment (mukhammadrizooff) [reply src/components/App.js:9]: - [ ] [REQUIRED] Your `Calculator` should matches this design:

calculator

- ### But you still haven't made the changes that the previous [code reviewer](https://github.com/nati2323/Math-Magicians/pull/2/files#r939552605) requested. Below your current design =>

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @nati2323, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] [REQUIRED] Please make the changes requested by previous code reviewers!

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @nati2323, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**